panels/user-accounts/cc-app-permissions.c

```diff
@@ -438,6 +440,26 @@ update_allow_app_installation (CcAppPermissions *self)
 {
   gboolean allow_system_installation;
   gboolean allow_user_installation;
+  gboolean non_admin_user = TRUE;
+
+  if (act_user_get_account_type (self->user) == ACT_USER_ACCOUNT_TYPE_ADMINISTRATOR)
+    non_admin_user = FALSE;
+
+  /* Admins are always allowed to install apps for all users. This behaviour is governed
+   * by flatpak polkit rules. Hence, these hide these defunct switches for admins. */
+  gtk_widget_set_visible (GTK_WIDGET (self->system_installation_label), non_admin_user);
+  gtk_widget_set_visible (GTK_WIDGET (self->allow_system_installation_switch), non_admin_user);
+
+  gtk_widget_set_visible (GTK_WIDGET (self->user_installation_label), non_admin_user);
+  gtk_widget_set_visible (GTK_WIDGET (self->allow_user_installation_switch), non_admin_user);
+
+  /* If user is admin, we are done here, bail out. */
+  if (!non_admin_user)
+    {
+      g_debug ("User %s is a admin user, hence app installation controls are defunct and hidden from UI",
```

Comment:
Nitpick: `"User %s is administrator, hiding app installation controls"`

Reply:
Addressed.